app/session/index.js

```diff
@@ -66,10 +66,6 @@ export function clear (request) {
   request.yar.clear(entries.tempClaimReference)
 }
 
-export function setClaim (request, key, value) {
-  set(request, entries.claim, key, value)
-}
-
 export function getClaim (request, key) {
```

Comment:
I see this (getClaim) is only referenced in the signin-oidc handler, which we shouldn't be using now anyway. But interesting that it will always return an undefined, and the logic that follows that get is all marked up with null guards on the info it tries to grab e.g organisation?.sbi and would always just lead to everything being undefined too. Don't need to do anything now, but we'll rip this out when that signin handler is finally binned off